tests/11-test-notification/confd.conf.part

```diff
@@ -0,0 +1,21 @@
+      <notification>
```

Comment:
feels a bit hacky assuming some meta-nested tag start position, dropping/overloading all the contents that may change order in original confd.conf, leading to loss of some config/params...